service/src/main/webapp/createUniversity.jsp

```diff
@@ -20,13 +20,6 @@
                 </div>
```

Comment:
createUniversity.jsp file name is inconsistent with the rest of the project.

Name should be */src/main/webapp/university/create.jsp* where university is a folder.